widget/ui/src/containers/Settings/Settings.tsx

```diff
@@ -111,7 +109,7 @@ export function Settings(props: PropTypes) {
     onSlippageChange(slippage);
   };
 
```

Comment:
Please move this `Settings` component out of ui package and declare it in `embedded`.